config/jobs/kubernetes-sigs/scheduler-plugins/scheduler-plugins-presubmits-release-1.18.yaml

```diff
@@ -0,0 +1,55 @@
+# sigs.k8s.io/scheduler-plugins presubmits
+presubmits:
+  kubernetes-sigs/scheduler-plugins:
+  - name: pull-scheduler-plugins-verify-gofmt-release-1.18
+    decorate: true
+    path_alias: sigs.k8s.io/scheduler-plugins
+    branches:
+    - ^release-1.18$
+    always_run: true
+    spec:
+      containers:
+      - image: golang:1.15.0
```

Comment:
(Maybe not quite matter) k/k:1.18 was using go 1.13:

https://github.com/kubernetes/kubernetes/blob/249a084b4eb6f5651b245c524ac63efb428ced28/go.mod#L7

Reply:
The container images for scheduler-plugins are built with Go 1.15.0.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/release-1.18/build/scheduler/Dockerfile#L14

But ... go.mod says 1.13.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/release-1.18/go.mod#L3

My opinion is stick with what is in the Dockerfile for schedler-plugins, but let me know if you want me to change it.